quantmsio/core/quantms/msstats_in.py

```
@@ -0,0 +1,702 @@
+                    self.logger.warning(f"Fallback column detection also failed: {e2}")
+                    # Default column set based on common msstats format
+                    available_columns = {
+                        "Reference",
+                        "ProteinName",
+                        "PeptideSequence",
+                        "Intensity",
+                    }
+
+            self._available_columns = available_columns
+
+        except Exception as e:
```

Flake8 (7.2.0)

[error] local variable 'e' is assigned to but never used

(F841)